admin_app/src/app/dashboard/components/TopicFiles.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+import { useAuth } from "@/utils/auth";
+import { Token } from "@mui/icons-material";
+
+interface Topic {
+  name: string;
+  count: number;
+  examples: { timestamp: string; userQuestion: string }[];
+}
+
+const TopicsContainer = styled.div`
```

Comment:
You aren't using any MUI components! Can you turn these all into MUI ones like using Box or Grid or Container.

https://mui.com/material-ui/react-box/